spec/models/apartment.rb

```diff
@@ -0,0 +1,4 @@
+class Apartment < ApplicationRecord
+    belongs_to :user
+    validates :street, presence: true
+  end
```

Comment:
I am not sure this file should be here. This shows that it is in the spec folder and there is already an apartment model.